src/main/java/com/RegisterQuizApplication/Registration/Choice.java

```diff
@@ -0,0 +1,40 @@
+package com.RegisterQuizApplication.Registration;
+
+public class Choice {
+    private String option_a;
```

Comment:
variable names should follow camelcasing. You can use @JsonProperty to distinguish the variable name and json property